package.json

```diff
@@ -0,0 +1,63 @@
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
+    "credentials": "npm run build && node dist/lib/update-credentials.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:projectSetup": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=true -DchannelsImport=true -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:catalogs": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=true -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:orders": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=false -DcatalogImport=false -DreserveInStore=false -DordersImport=true -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:zones": "npm run build && node dist/lib/zones.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "import:shippingmethods": "npm run build && node dist/lib/shipping-method.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "sync:products": "node_modules/.bin/product-csv-sync import --matchBy sku --csv data/products.csv --projectKey $npm_package_config_project --clientId $npm_package_config_client_id  --clientSecret $npm_package_config_client_secret --sphereProtocol https --sphereAuthProtocol https",
```

Comment:
The products import works, but complains about:
```
CSV file with 26138 row(s) loaded.
For the product type 'main' the following attributes don't have a matching header:
  sapExternalId: type 'text ' - constraint 'Unique' - optional
```
I guess this needs to go into a new data model issues

Reply:
This is the only real bug:  the products are all unpublished and will therefore not appear in a storefront like SUNRISE.   You need to set the necessary flag on the product syncer.

Reply:
Yes, created a new ticket for missing header attribute #102

product publish flag is added. 8cb8ad6